src/components/cesium_map/cesium_UI.js

```diff
@@ -26,8 +26,8 @@ import { addButton } from "./elements/navigationButton";
 Cesium.Ion.defaultAccessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIwNzk3NjkyMy1iNGI1LTRkN2UtODRiMy04OTYwYWE0N2M3ZTkiLCJpZCI6Njk1MTcsImlhdCI6MTYzMzU0MTQ3N30.e70dpNzOCDRLDGxRguQCC-tRzGzA-23Xgno5lNgCeB4';
 
 // constant variables
-const REFRESH_TIME_MS = 5000;
-const VIEWPOINT_TIME_MS = 2000;
+const REFRESH_TIME_MS = 30000;
+const VIEWPOINT_TIME_MS = 30000;
 const UPDATE_RATIO = 0.5;
```

Comment:
Increment the refresh and viewpoint save time. The refresh and viewpoint interval function is called even before the entire points are loaded, causing out of memory issues.